graphdatascience/graph_data_science.py

```diff
@@ -55,5 +55,5 @@ def from_neo4j_driver(cls: Type[GDS], driver: Driver) -> "GraphDataScience":
         return cls(cls.create_neo4j_query_runner(driver))
 
     @staticmethod
-    def create_neo4j_query_runner(driver: Driver) -> Neo4jQueryRunner:
-        return Neo4jQueryRunner(driver)
+    def create_neo4j_query_runner(driver: Driver, auto_close: bool = False) -> Neo4jQueryRunner:
+        return Neo4jQueryRunner(driver, auto_close=auto_close)
```

Comment:
```suggestion
        return Neo4jQueryRunner(driver, auto_close = auto_close)
```

Reply:
not sure what your overall code style is wrt default parameters, but it should be just one :)

Reply:
These are two different things: _defining_ keyword parameters, and _using_ keyword parameters. For each there are very strict rules which are enforced by our most opinionated formatter https://black.readthedocs.io/en/stable/. There's no conflict of styles here.

Reply:
Thanks for the explanation. My Python is Rust-y *hue hue*